__tests__/app.test.js

```diff
@@ -68,11 +90,45 @@ describe("app", () => {
               article_img_url: expect.any(String),
               comment_count: expect.any(Number),
             });
-            expect(articles).toBeSortedBy('created_at', {
-                descending: true
+            expect(articles).toBeSortedBy("created_at", {
+              descending: true,
             });
           });
         });
     });
   });
+  describe("GET /api/articles/:article_id/comments", () => {
+    it("responds with an array", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+          const comments = body;
+          expect(Array.isArray(comments)).toBe(true);
+          expect(comments.length).toBeGreaterThan(0);
+        });
+    });
+    it("responds with an array of comments for specific article_id, most recent comment first", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+          const comments = body;
+          comments.forEach(comment => {
+            expect(comment).toMatchObject({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              article_id: expect.any(Number),
+            });
+          })
+
+          expect(comments).toBeSortedBy('created_at', {
+            descending: true
+          })
+        });
```

Comment:
I would add another 200 test for an article that exists but it has no comments - to check that we get an empty array returned.